JS/Lesson16/esra/script.js

```diff
@@ -62,7 +65,7 @@ switch (day) {
    break;
   case 'Tuesday':
     console.log('It\'s Tuesday');
-    break; // Use break key word if you don't use return key word in the statement;
+    break; // Use break key word if you don't useconsole.log(key word in the statement;
```

Comment:
This is not correct! `Console.log` statement does not stop the execution of the code in the switch statement. Should be: "// Use break key word if you don't use return key word in the statement;"

Reply:
```suggestion
    break; // Use break key word if you don't use return key word in the statement;
```

Reply:
The change was made and committed from your computer, that's what Git tells here :D 

What I mean that it should be changed to the initial version. I see that you did it already so nothing more to fix here :)